### PR TITLE
feat(api): add resource price API for solidity and PBFT  

### PR DESCRIPTION
**Pull request by lxcmyf**
**What does this PR do?**
add resource unit price inquiry API for Solidity and PBFT, close https://github.com/tronprotocol/java-tron/issues/5437
**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: framework/src/test/java/org/tron/core/services/interfaceOnSolidity/http/GetBandwidthPricesOnSolidityServletTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import org.tron.core.Constant;
+import org.tron.core.config.args.Args;
+
+public class GetBandwidthPricesOnSolidityServletTest extends BaseTest {
+
+  @Resource
+  private GetBandwidthPricesOnSolidityServlet getBandwidthPricesOnSolidityServlet;
+
+  static {
+    dbPath = "db_GetBandwidthPricesOnSolidityServlet_test";
```
Comment (halibobo1205) [inline framework/src/test/java/org/tron/core/services/interfaceOnSolidity/http/GetBandwidthPricesOnSolidityServletTest.java:25]:
```java
        @Rule
        public TemporaryFolder folder= new TemporaryFolder();
```
TemporaryFolder is recommended.

Comment (lxcmyf) [reply framework/src/test/java/org/tron/core/services/interfaceOnSolidity/http/GetBandwidthPricesOnSolidityServletTest.java:25]: Let me give it a try.

File: framework/src/test/java/org/tron/core/services/interfaceOnPBFT/http/GetEnergyPricesOnPBFTServletTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.tron.common.BaseTest;
+import org.tron.core.Constant;
+import org.tron.core.config.args.Args;
+
+public class GetEnergyPricesOnPBFTServletTest extends BaseTest {
```
Comment (halibobo1205) [inline framework/src/test/java/org/tron/core/services/interfaceOnPBFT/http/GetEnergyPricesOnPBFTServletTest.java:23]: Meaningless messages will be output.
image

Comment (lxcmyf) [reply framework/src/test/java/org/tron/core/services/interfaceOnPBFT/http/GetEnergyPricesOnPBFTServletTest.java:23]: OK，I'll remove it.

File: framework/src/test/java/org/tron/common/BaseTest.java [hunk 3]
```diff
@@ -39,8 +39,6 @@ public static void destroy() {
     Args.clearParam();
     if (StringUtils.isNotEmpty(dbPath) && FileUtil.deleteDir(new File(dbPath))) {
```
Comment (halibobo1205) [inline framework/src/test/java/org/tron/common/BaseTest.java:40]: Suggested change :
```java
if (StringUtils.isNotEmpty(dbPath)) {
    if (FileUtil.deleteDir(new File(dbPath))) {
        logger.info("Release resources successful.");
     } else {
        logger.info("Release resources failure.");
     }
 }
```

Comment (lxcmyf) [reply framework/src/test/java/org/tron/common/BaseTest.java]: Okay, synchronized.

**Review by wubin01 — APPROVED** [review]

**Review by chengtx01 — APPROVED** [review]

**Pull request merged**